docs/build-with-php.md

```diff
@@ -0,0 +1,231 @@
+UN  10.10.5.3  1.06 MB    256          ?       871795f3-67d2-47ba-83ef-15714b89c02a  rack1
+UN  10.10.5.4  1.06 MB    256          ?       cbe74a63-2cf4-41c2-bf7f-c831c0d2689f  rack1
+`````
+
+##### Running Container Shell:
+
+````shell
+$ docker exec -it carepet-scylla1 bash
+
+   _____            _ _       _____  ____
+  / ____|          | | |     |  __ \|  _ \
+ | (___   ___ _   _| | | __ _| |  | | |_) |
+  \___ \ / __| | | | | |/ _` | |  | |  _ <
+  ____) | (__| |_| | | | (_| | |__| | |_) |
+ |_____/ \___|\__, |_|_|\__,_|_____/|____/
+               __/ |
+              |___/
+Nodetool:
+        nodetool help
+CQL Shell:
+        cqlsh
+More documentation available at:
+        http://www.scylladb.com/doc/
+
+root@7e2b1b94389b:/#
+````
+##### Inspecting a Container
+_You can inspect any node by means of the `docker inspect` command as follows. For example:_
+
+````shell
+$ docker inspect carepet-scylla1
+[
+    {
+        "Id": "7e2b1b94389b36c494093db8e119c2b8c5167339f20e03d9bfa070e8e46f8430",
+        "Created": "2023-01-05T17:36:59.038609825Z",
+        "Path": "/docker-entrypoint.py",
+        "Args": [
+            "--smp",
+            "1"
+        ],
+        "State": {
+            "Status": "running",
+            "Running": true,
+            "Paused": false,
+            "Restarting": false
+            ...
+        }
+    }
+]
+````
+
+###### Get Node IP Address:
+````shell
+$ docker inspect -f '{{range .NetworkSettings.Networks}}{{.IPAddress}}{{end}}' carepet-scylla1
+10.10.5.2
+````
+
+##### Entering CQLSH (Database)
+
+````shell
+$ docker exec -it carepet-scylla1 cqlsh
+Connected to  at 10.10.5.2:9042.
+[cqlsh 5.0.1 | Cassandra 3.0.8 | CQL spec 3.3.1 | Native protocol v4]
+Use HELP for help.
+cqlsh>
+````
+
+````sql
+cqlsh> DESCRIBE KEYSPACES
+carepet  system_schema  system_auth  system  system_distributed  system_traces
+````
+
+````sql
+cqlsh> USE carepet;
+cqlsh:carepet> DESCRIBE TABLES
+pet  sensor_avg  gocqlx_migrate  measurement  owner  sensor
+````
+````sql
+cqlsh:carepet> DESCRIBE TABLE pet
+CREATE TABLE carepet.owner (
+    owner_id uuid PRIMARY KEY,
+    address text,
+    name text
+) WITH bloom_filter_fp_chance = 0.01
+    AND caching = {'keys': 'ALL', 'rows_per_partition': 'ALL'}
+    AND comment = ''
+    AND compaction = {'class': 'SizeTieredCompactionStrategy'}
+    AND compression = {'sstable_compression': 'org.apache.cassandra.io.compress.LZ4Compressor'}
+    AND crc_check_chance = 1.0
+    AND dclocal_read_repair_chance = 0.0
+    AND default_time_to_live = 0
+    AND gc_grace_seconds = 864000
+    AND max_index_interval = 2048
+    AND memtable_flush_period_in_ms = 0
+    AND min_index_interval = 128
+    AND read_repair_chance = 0.0
+    AND speculative_retry = '99.0PERCENTILE';
+
+cqlsh:carepet> exit
+````
+
+## Architecture
+
+Pet --> Sensor --> ScyllaDB <-> REST API Server <-> User
+
+- https://hub.docker.com/r/scylladb/scylla
+- https://github.com/datastax/php-driver

```

Comment:
This links does not render nicely in Sphinx
![image](https://user-images.githubusercontent.com/170200/221801777-a8051531-1080-4dbd-a1a8-6dca3fd767cf.png)